controller/auth/manager.py

```diff
@@ -152,3 +154,34 @@ def extract_state_info(request: Request, key: str) -> Any:
         return value
 
     return request.state.parsed[key]
+
+
+def check_is_full_admin(request: Any) -> bool:
+    if check_is_admin(request):
+        jwt_decoded: Dict[str, Any] = jwt.decode(
+            request.headers["Authorization"].split(" ")[1],
+            options={"verify_signature": False},
+        )
+        subject: Dict[str, Any] = jwt_decoded["session"]["identity"]
+        if check_email_in_full_admin(subject["traits"]["email"]):
+            return True
+    return False
+
```

Comment:
can we add a check_invite_users function? I'm thinking of ensureing the email addresses are following a common email regex & dont have an account yet in kratos so if someone pastes an already existing we dont send. On the frontent the button for inviting should only be enabled if the check was successful

Reply:
yes sure, i will add this

Reply:
should this be in its own endpoint, that is called on 'Continue' from the modal ?

Reply:
sounds good :)